packages/string/src/utils/types.ts

```
@@ -0,0 +1,68 @@
+     * @param text - The text to measure
+     * @returns The visual width
+     */
+    getWidth?: (text: string) => number;
+
+    /**
+     * Function to process each ANSI escape sequence
+     * @param sequence - The ANSI sequence
+     * @param stateTracker - The ANSI state tracker
+     * @returns Whether to continue processing or not
+     */
+    onEscapeSequence?: (sequence: string, stateTracker: AnsiStateTracker) => boolean | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)